.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig http://EditorConfig.org
+
+# Project Root
+root = true
+
+# Default Code Style
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+tab_width = 4
```

Comment:
I'm talking about https://docs.python.org/2/reference/lexical_analysis.html#indentation

    First, tabs are replaced (from left to right) by one to eight spaces such that the total number of
    characters up to and including the replacement is a multiple of eight (this is intended to be the
    same rule as used by Unix)

So setting `tab_width = 4` on `[*.py]` is simply wrong (as inherited from `[*]`), and setting `tab_width = 4` on `[*]` is unnecessary because none of our files use tabs, so let's just remove it.